qa-physics/epochs/epochs.rgm_fall2021.txt

```diff
@@ -0,0 +1,21 @@
+15019 15199
+15200 15311
+15312 15317
+15318 15327
+15356 15390
+15391 15434
+15435 15456
+15458 15490
+15542 15563
+15566 15627
+15628 15636
+15637 15638
+15643 15670
+15671 15724
+15726 15727
+15733 15733
+15739 15764
+15766 15784
+15789 15802
+15803 15827
+15830 15884
```

Comment:
@esteejus are these the epochs that you are using?

Reply:
```suggestion
15019 15199
15200 15317
15318 15327
15356 15389
15390 15432
15434 15456
15458 15490
15542 15563
15566 15627
15628 15636
15637 15670
15671 15727
15733 15764
15766 15784
15789 15802
15803 15827
15830 15884
```

Reply:
Hi Chris, it should be the above suggested edits.